benchmarks/benchmark_serving.py

```diff
@@ -359,6 +525,27 @@ def main(args: argparse.Namespace):
         default=1000,
         help="Number of prompts to process.",
     )
+    parser.add_argument(
+        "--input-len",
+        type=int,
+        default=550,
+        help=
+        "Number of input tokens per request, used only for sonnet dataset.",
+    )
+    parser.add_argument(
+        "--output-len",
+        type=int,
+        default=150,
+        help=
+        "Number of output tokens per request, used only for sonnet dataset.",
+    )
+    parser.add_argument(
+        "--prefix-len",
+        type=int,
+        default=200,
+        help=
+        "Number of prefix tokens per request, used only for sonnet dataset.",
+    )
```

Comment:
name them --sonnet-input-len, ... etc because the names are so generic that I can imagine anyone will misuse them